src/glum/_glm.py

```diff
@@ -452,7 +452,7 @@ def _one_over_var_inf_to_val(arr: np.ndarray, val: float) -> np.ndarray:
 
     If values are zeros, return val.
     """
-    zeros = np.where(np.abs(arr) < 1e-7)
+    zeros = np.where(np.abs(arr) < np.sqrt(np.finfo(arr.dtype).eps))
```

Comment:
@mlondschien, what was the logic for this change? It's a slightly stricter criterion for `float64` than before, which is causing testing failures downstream.

Reply:
```
In [4]:  np.sqrt(np.finfo(np.float32).eps)
Out[4]: np.float32(0.00034526698)

In [5]:  np.sqrt(np.finfo(np.float64).eps)
Out[5]: np.float64(1.4901161193847656e-08)
```
None. I thought that's where the 1e-7 came from. I don't realy know what would be "good" values here. Just that 1e-7 is too strict for `float32`. Sorry!